app/experimenter/visualization/tests/api/constants.py

```diff
@@ -1,371 +1,204 @@
-                "statistic": "mean",
-                "branch": "variant",
-            },
-        }
-        VARIANT_DATA_DEFAULT_METRIC_ROW_BINOMIAL = {
-            **DATA_IDENTITY_ROW,
-            **{
-                "metric": "another_count",
-                "statistic": "binomial",
-                "branch": "variant",
-            },
-        }
-        VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW = {
-            **DATA_IDENTITY_ROW,
-            **{
-                "comparison": "difference",
-                "metric": "search_count",
-                "statistic": "mean",
-                "branch": "variant",
-            },
-        }
-        VARIANT_NEGATIVE_SIGNIFICANCE_DATA_ROW = {
-            **VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW,
-            **{
-                "point": -2,
-                "upper": -1,
-                "lower": -5,
-                "metric": "retained",
-                "statistic": "binomial",
-            },
-        }
-        CONTROL_NEUTRAL_SIGNIFICANCE_DATA_ROW = {
-            **VARIANT_NEGATIVE_SIGNIFICANCE_DATA_ROW,
-            **{
-                "point": 12,
-                "upper": 13,
-                "lower": -5,
-                "branch": "control",
-            },
-        }
-        DATA_WITHOUT_POPULATION_PERCENTAGE = [
-            CONTROL_DATA_ROW,
-            VARIANT_DATA_ROW,
-            VARIANT_DATA_DEFAULT_METRIC_ROW_MEAN,
-            VARIANT_DATA_DEFAULT_METRIC_ROW_BINOMIAL,
-            VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW,
-            VARIANT_NEGATIVE_SIGNIFICANCE_DATA_ROW,
-            CONTROL_NEUTRAL_SIGNIFICANCE_DATA_ROW,
+        DATA_IDENTITY_ROW = JetstreamDataPoint(
+            point=12,
+            upper=13,
+            lower=10,
+            metric="identity",
+            statistic="count",
+            window_index="1",
+        )
+
+        CONTROL_DATA_ROW = DATA_IDENTITY_ROW.copy()
+        CONTROL_DATA_ROW.branch = "control"
+
+        VARIANT_DATA_ROW = DATA_IDENTITY_ROW.copy()
+        VARIANT_DATA_ROW.branch = "variant"
+
+        VARIANT_DATA_DEFAULT_METRIC_ROW_MEAN = DATA_IDENTITY_ROW.copy()
+        VARIANT_DATA_DEFAULT_METRIC_ROW_MEAN.metric = "some_count"
+        VARIANT_DATA_DEFAULT_METRIC_ROW_MEAN.statistic = "mean"
+        VARIANT_DATA_DEFAULT_METRIC_ROW_MEAN.branch = "variant"
+
+        VARIANT_DATA_DEFAULT_METRIC_ROW_BINOMIAL = DATA_IDENTITY_ROW.copy()
+        VARIANT_DATA_DEFAULT_METRIC_ROW_BINOMIAL.metric = "another_count"
+        VARIANT_DATA_DEFAULT_METRIC_ROW_BINOMIAL.statistic = "binomial"
+        VARIANT_DATA_DEFAULT_METRIC_ROW_BINOMIAL.branch = "variant"
+
+        VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW = DATA_IDENTITY_ROW.copy()
+        VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW.comparison = "difference"
+        VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW.metric = "search_count"
+        VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW.statistic = "mean"
+        VARIANT_POSITIVE_SIGNIFICANCE_DATA_ROW.branch = "variant"
+
```

Comment:
Maybe some of these strings like `"mean"` `"binomial"` etc could be constants so it's clear which values are arbitrary data like branch names and which are expected constants?